src/com/cft/contactmerge/contact/Address.java

```diff
@@ -0,0 +1,96 @@
+package com.cft.contactmerge.contact;
+
+public class Address {
+    String streetAddress;
+    String apartment;
+    String city;
+    String state;
+    String country;
+    String zip;
+    String addressType;
+
+    /*******************************************************************************************************************
+     *************************************************** Constructors **************************************************
+     *******************************************************************************************************************/
+    public Address(){
+        setStreetAddress("");
+        setApartment("");
+        setCity("");
+        setState("");
+        setCountry("");
+        setZip("");
+        setAddressType("");
+    }
+
+    public Address(String streetAddress, String apartment, String city, String state, String country, String zip, String addressType){
```

Comment:
FYI... I was thinking that AddressType, PhoneType, and EmailType might be enums. Not something we need in the initial release so could just leave them out completely until we figure out how was are going to deal with them. Sample of AddressType: Home Address, Business Address, Seasonal Address, Previous Address. Sample of PhoneType: Home Phone, Mobile Phone, Business Phone, Previous Phone. Sample of EmailType: Personal Email; Business Email; Previous Email. We will also (eventually) need to represent the concept of "preferred contact methods". GiftWorks does this by letting you pick a primary, secondary and tertiary address/email/phone. Again, not something we have to worry about now since we only have one of each.

Reply:
Okay, I will remove them for now.